hudi-utilities/src/main/java/org/apache/hudi/utilities/config/HoodieIncrSourceConfig.java

```diff
@@ -101,4 +101,11 @@ public class HoodieIncrSourceConfig extends HoodieConfig {
       .withAlternatives(DELTA_STREAMER_CONFIG_PREFIX + "source.hoodieincr.partition.extractor.class")
       .markAdvanced()
       .withDocumentation("PartitionValueExtractor class to extract partition fields from _hoodie_partition_path");
+
+  public static final ConfigProperty<String> HOODIE_SPARK_DATASOURCE_OPTIONS = ConfigProperty
+      .key(STREAMER_CONFIG_PREFIX + "source.hoodieincr.data.datasource.options")
+      .noDefaultValue()
+      .markAdvanced()
+      .withDocumentation("A comma separate list of options that can be passed to the spark dataframe reader of a hudi table, "
+          + "eg: hoodie.metadata.enable=true,hoodie.enable.data.skipping=true");
```

Comment:
```suggestion
      .withDocumentation("A comma-separated list of Hudi options that can be passed to the spark dataframe reader of a hudi table, "
          + "eg: `hoodie.metadata.enable=true,hoodie.enable.data.skipping=true`. Used only for incremental source.");
```

Reply:
Btw, if this is a useful config for other sources too, then move to `HoodieReaderConfig` and rename config variable and key accordingly.

Reply:
We can keep the config in the `HoodieIncrSourceConfig` class since it applies to the incremental source only.